src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1265,8 +1265,11 @@ private async Task ValidateUnusedIdsAsync(HashSet<string> unusedIds, string data
         {
             foreach (var id in unusedIds)
             {
-                if(IsBase64String(id)==false)
-                    throw new InvalidOperationException($"Database id '{id}' isn't valid because it isn't Base64String (it contains chars which cannot be in Base64String).");
+                if (id.Length != 22)
```

Comment:
introduce a const and use it in both places, so there will be no surprises in the future if someone will change that

Reply:
Done